plugins/com.google.cloud.tools.eclipse.dataflow.core/src/com/google/cloud/tools/eclipse/dataflow/core/project/DataflowProjectCreator.java

```diff
@@ -347,8 +347,10 @@ private DataflowProjectValidationStatus validateMavenGroupId() {
   }
 
   private DataflowProjectValidationStatus validatePackage() {
-    if (Strings.isNullOrEmpty(packageString)
-        || !JAVA_PACKAGE_REGEX.matcher(packageString).matches()) {
+    if (Strings.isNullOrEmpty(packageString)) {
+      return DataflowProjectValidationStatus.OK;
```

Comment:
Previously, if it's null or empty, it returned `ILLEGAL_PACKAGE`, which is an error. Now you are allowing it. Can you verify there will be no issue when creating a project without a package name?

But even if is OK, I realized a little technical problem. If you ever enter anything into the package field (which will always happen the field is auto-filled when you enter a Maven group ID), you cannot make the field empty: What happens is that, if you delete the auto-filled value in the package field, the value is automatically restored as soon as the focus is lost from the field.

Maybe we should just return `.MISSING`.